include/sdf/Cylinder.hh

```diff
@@ -72,6 +75,12 @@ namespace sdf
     /// \return A reference to a gz::math::Cylinderd object.
     public: gz::math::Cylinderd &Shape();
 
+    /// \brief Calculate and return the Mass Matrix values for the Cylinder
+    /// \param[in] density Density of the cylinder in kg/m^3
```

Comment:
```suggestion
    /// \param[in] _density Density of the cylinder in kg/m^3
```

Reply:
Corrected in [1f16e1f](https://github.com/gazebosim/sdformat/pull/1304/commits/1f16e1f0a5454acc5a30a4b86b8f273b1535eb95)